client/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,14 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { DashboardComponent } from './dashboard/dashboard.component';
+import { LoginComponent } from './login/login.component';
+import { UserProfileComponent } from './user-profile/user-profile.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'dashboard', component: DashboardComponent },
+  { path: 'login', component: LoginComponent },
+  { path: 'user-profile', component: UserProfileComponent },
+];
```

Comment:
Eventually there will be only two pages, Dashboard and Advice wall. Other will be a button with a modal pop up window. You can remove login and user profile from the route. But make a buttons: Mentors, Mentorship, Bell icon, button with the username, logout button.

So combining these pages and button will look as follows:
Dashboard, Advice Wall, Mentors, Mentorship, bell icon, Aisulu Abdraimova, logout button.